### ISSUE TITLE
Questions about setting Client ID for dot

### ISSUE THREAD

**Issue by liu-han11** [AdguardTeam/AdGuardHome#2673]
Thank you very much for answering my question here.
My problem is that when I set the Client ID for the dot, my secondary domain is one.example.com,
The format of dot is tls://one.example.com,
The ssl certificate domain is *.example.com.
When I set the Client ID, the domain name will be tls://ClientID.one.example.com, but it doesn't match the *.example.com in the SSL certificate, I don't know how to solve it.
I looked for a lot of ways, but there is no way to solve this problem, maybe I did not understand the principle of the configuration myself.
So please ask you, how should I set it so that I can use dot's Client ID?
Finally, I feel very grateful for your answer, and I thank you again.

**Comment by ainar-g** [comment]
Hello.  What is the server name you put in on the “Encryption settings” page?

**Comment by ainar-g** [comment]
Also, please check the solution in #2672.

**Comment by liu-han11** [comment]
I just saw your reply and the way to answer it in #2672. It turned out to be a problem with my ssl certificate application. I used certbot to apply for a certificate, and based on the tips above, I set the statement to apply for an ssl certificate to
`certbot certonly -d one.example.com -d *.one.example.com --manual \--preferred-challenges dns \--server https://acme-v02.api. letsencrypt.org/directory \--preferred-chain "ISRG Root X1"`
This solved my problem.
Thank you, and thanks for your answer in #2672.
Again, thank you very much.
